pkg/wrap.go

```diff
@@ -111,20 +111,20 @@ func cAccept(s C.SRTSOCKET, addr *C.struct_sockaddr, size *C.int) (C.SRTSOCKET,
 	return ret, nil
 }
 
-func cListenCallback(lsn C.SRTSOCKET) error {
+func cListenCallback(lsn *C.SRTSOCKET) error {
```

Comment:
the `wrap.go` file is auto-generated therefore you need to change the `internal/cmd/generate/wrap/main.go` file. For instance, for `cListenCallback` you need to do the following changes :

- [here](https://github.com/asticode/go-astisrt/blob/master/internal/cmd/generate/wrap/main.go#L385) it needs to be
```go
{
	Format: "*%s",
	Name:   "lsn",
	Type:   "*C.SRTSOCKET",
},
```
- and [here](https://github.com/asticode/go-astisrt/blob/master/internal/cmd/generate/wrap/main.go#L389) it needs to be:
```go
{
	// We need to provide a GO pointer otherwise, if we provide a C
	// pointer later, exported go function sometimes doesn't retrieve
	// the proper value
	Name: "unsafe.Pointer(lsn)",
},
```

Changes for `cConnectCallback` are similar.

Then to generate `wrap.go` you can use `go run internal/cmd/generate/wrap/main.go`